tests/Areas/Cosmos/LiveTests/CosmosCommandTests.cs

```diff
@@ -20,7 +20,7 @@ public class CosmosCommandTests(LiveTestFixture liveTestFixture, ITestOutputHelp
     public async Task Should_list_storage_accounts_by_subscription_id()
     {
         var result = await CallToolAsync(
-            "azmcp-cosmos-database-list",
+            "azmcp_cosmos_database_list",
```

Comment:
This will need to be updated in all tests.

Reply:
yep. 'cause I did not setup live test config in my local, I fixed some of them and push one iteration to check whether these live tests are fixed in pipeline. then I will fix rest of them in the next iteration.